Python/Vector Calculator/readme.md

```diff
@@ -0,0 +1,43 @@
+# Vector Calculator
+
+## Description
+
+Vector Calculator GUI based app built using Python that can fetch relation between 2 vectors along with the properties of each vector alone.
+
+Relations that can be calculated:
+
+* Dot product.
+* Cross product.
+* Angle between both vectors.
+* Component of each vector on the other.
+* Projection of each vector on the other.
+
+Properties that can be calculated:
+
+* Unit Direction of each vector.
+* Cosine Direction of each vector.
+* Cosine Angles of each vector.
+* Absolute Value of each vector. |A|
+
+## Tech Stack
+
+* Python3 -> Tkinter
+
+## Install the Dependencies
+
+* Install Python 3.6 or above.
+* Further install these libraries using terminal:
+
+```bash
+pip3 install tkinter
+```
+
+## Run Local
+
+```bash
+python3 calc.py
+```
```

Comment:
@XZANATOL please try to mention in these way and if possible then try to add Demo video or gif.
<img width="1080" alt="Screenshot 2022-03-03 at 1 01 42 PM" src="https://user-images.githubusercontent.com/78725162/156778327-c3184881-db60-48cb-b954-2d585741f0af.png">